### ISSUE TITLE
"Found pixel at #" error

### ISSUE THREAD

**Issue by kckunal2612** [romannurik/AndroidAssetStudio#82]
I am using Android Studio 1.5, with the build tools version 23.0.2, and am getting a Gradle Build Error for every 9-patch image that I use (using your tool). 
Here is the complete error message from GradleBuild Log -

`AAPT: ERROR: 9-patch image /home/kunal/AndroidStudioProjects/MyApplication/app/src/main/res/drawable-hdpi/munchery_ondemand.9.png malformed.`

`AAPT:  Frame pixels must be either solid or transparent (not intermediate alphas).`
`AAPT:        Found at pixel #360 along bottom edge.`
`AAPT: ERROR: 9-patch image /home/kunal/AndroidStudioProjects/MyApplication/app/src/main/res/drawable-xhdpi/munchery_ondemand.9.png malformed.`
`AAPT:        Frame pixels must be either solid or transparent (not intermediate alphas).`
`AAPT:        Found at pixel #480 along bottom edge.`
`AAPT err(Facade for 1648474378) : No Delegate set : lost message:ERROR: 9-patch image /home/kunal/AndroidStudioProjects/MyApplication/app/src/main/res/drawable-xxxhdpi/munchery_ondemand.9.png malformed.`
`AAPT err(Facade for 1648474378) : No Delegate set : lost message:       Frame pixels must be either solid or transparent (not intermediate alphas).`
`AAPT err(Facade for 1648474378) : No Delegate set : lost message:       Found at pixel #959 along top edge.`
`:app:mergeDebugResources FAILED`
`Error:Execution failed for task ':app:mergeDebugResources'.`
`> Some file crunching failed, see logs for details`
`Information:BUILD FAILED`

**Comment by mukesh4u** [comment]
Add below line inside build.gradle
aaptOptions.setProperty("cruncherEnabled", false)

**Comment by dandar3** [comment]
Same problem here with drawables generated by **Simple Nine-patch Generator**.
Oddly enough the issue only happens for xhdpi, xxhdpi, xxxhdpi (?!)

Android Studio 2.1.2
Android Plugin for Gradle 2.1.2
Build Tools 24.0.0

```:app:mergeDebugResources
Executing task ':app:mergeDebugResources' (up-to-date check took 0.078 secs) due to:
  Input file R:\Workspaces\Android\[...]\app\src\main\res\drawable-xhdpi\ic_logo_splash.9.png has been added.
  Input file R:\Workspaces\Android\[...]\app\src\main\res\drawable-xxxhdpi\ic_logo_splash.9.png has been added.
  Input file R:\Workspaces\Android\[...]\app\src\main\res\drawable-mdpi\ic_logo_splash.9.png has been added.
QueuedCruncher is using D:\Android\android-sdk\build-tools\24.0.0\aapt.exe

AAPT err(Facade for 193471613): ERROR: 9-patch image R:\Workspaces\Android\[...]\app\src\main\res\drawable-xhdpi\ic_logo_splash.9.png malformed.
AAPT err(Facade for 193471613):        Frame pixels must be either solid or transparent (not intermediate alphas).
AAPT err(Facade for 193471613):        Found at pixel #284 along bottom edge.
AAPT err(Facade for 193471613): ERROR: 9-patch image R:\Workspaces\Android\[...]\app\src\main\res\drawable-xxxhdpi\ic_logo_splash.9.png malformed.
AAPT err(Facade for 193471613):        Frame pixels must be either solid or transparent (not intermediate alphas).
AAPT err(Facade for 193471613):        Found at pixel #287 along bottom edge.
AAPT err(Facade for 96010134): ERROR: 9-patch image R:\Workspaces\Android\[...]\app\src\main\res\drawable-xxhdpi\ic_logo_splash.9.png malformed.
AAPT err(Facade for 96010134):        Frame pixels must be either solid or transparent (not intermediate alphas).
AAPT err(Facade for 96010134):        Found at pixel #504 along right edge.

:app:mergeDebugResources FAILED
```

Setting AAPT `cruncherEnabled` to `false` doesn't seem to have an effect for me.

**Comment by dandar3** [comment]
@mukesh4u 
FYI It appears AAPT cruncher setting applies to non-9patch only, see below:
https://medium.com/@duhroach/smaller-pngs-and-android-s-aapt-tool-4ce38a24019d#.647noh4n1

> ```
> aaptOptions {
>  cruncherEnabled = false
>  }
> ```
> 
> This will turn off the AAPT png optimization for non-9patch PNG files inside of your APK (I haven’t figured out what it does with 9Patches yet…)

**Comment by romannurik** [comment]
Please re-open if this is still a problem (and attach an example file that's causing the errors); I'm not seeing any errors.

**Comment by localhostEU** [comment]
Having same error. Please reopen.

**Comment by rtuck99** [comment]
This is indeed a bug with Android Asset Studio. Import the attached xxxhdpi source file, set regions and then export and get the attached zip. The xxxhdpi file produces the error:

AAPT err(Facade for 1513812613) : No Delegate set : lost message:ERROR: 9-patch image /home/bob/workspace/cards/cards-android/app/src/main/res/drawable-xxxhdpi/medium_button.9.png malformed.
AAPT err(Facade for 1513812613):        Frame pixels must be either solid or transparent (not intermediate alphas).
AAPT err(Facade for 1513812613):        Found at pixel #125 along top edge.

Inspect pixel 125 and you will see it indeed has alpha 0x07

![medium_button](https://user-images.githubusercontent.com/4296093/29370960-de6c01ea-829e-11e7-8afa-b8ff68ec18df.png)
[medium_button.9 (7).zip](https://github.com/romannurik/AndroidAssetStudio/files/1228646/medium_button.9.7.zip)

**Comment by romannurik** [comment]
Should be fixed by a4ddd69d987fbbb24a5012418f07b019d611b136
